example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -278,33 +279,43 @@ describe('The S3 Ingest Granules workflow', () => {
     ]);
   });
 
+  beforeEach(() => {
+    if (beforeAllError) fail(beforeAllError);
+  });
+
   it('prepares the test suite successfully', () => {
-    if (beforeAllFailed) fail('beforeAll() failed to prepare test suite');
+    if (beforeAllError) fail(`beforeAll() failed to prepare test suite: ${testSetupError}`);
   });
 
   it('triggers a running execution record being added to DynamoDB', async () => {
+    if (beforeAllError) throw SetupError;
+
     const record = await waitForModelStatus(
       executionModel,
       { arn: workflowExecutionArn },
-      'running'
+      ['running', 'completed']
     );
     expect(record.status).toEqual('running');
   });
 
   it('triggers a running PDR record being added to DynamoDB', async () => {
+    if (beforeAllError) throw SetupError;
+
     const record = await waitForModelStatus(
       pdrModel,
       { pdrName: inputPayload.pdr.name },
-      'running'
+      ['running', 'completed']
```

Comment:
This is actually the 'guts' of the change here - under load in Bamboo/local CI, occasionally we'd wind up with the write in a completed state and the test waiting until the timeout.   

In inspection, it didn't appear there were database write failures on the running status.